core/src/main/java/com/linecorp/armeria/common/HttpStatus.java

```diff
@@ -491,6 +492,48 @@ public boolean isContentAlwaysEmpty() {
         return isContentAlwaysEmpty(code);
     }
 
+    /**
+     * Returns whether the {@link HttpStatus} is an information, with a status code of 1XX.
+     */
+    public boolean isInformational() {
+        return codeClass().equals(HttpStatusClass.INFORMATIONAL);
```

Comment:
This may be more efficient:

    return HttpStatusClass.INFORMATIONAL.contains(code);